_posts/2021-05-14-bring-gitops-to-your-functions-with-argocd-part-2-the-missing-piece-tekton.md

```diff
@@ -0,0 +1,612 @@
+---
+title: "Bring GitOps to your OpenFaaS functions with Argo CD - The missing piece of the pipeline: Tekton"
+description: "In this post you'll learn how to combine these two technologies ArgoCD and Tekton to build fully automated CI/CD pipeline which can be triggerred by Github to manage OpenFaaS functions based on GitOps principles in your local environment using a Cloud Native Tunnel called Inlets"
+date: 2021-05-14
+image: /images/bring-gitops-to-your-functions-with-argocd-part-2-the-missing-piece-tekton/tekton_argocd_power.png
+categories:
+- arkade
+- kubectl
+- argocd
+- tekton
+- gitops
+- openfaas-operator
+- helm
+author_staff_member: developer-guy
+dark_background: true
+
+---
+
+In this post you'll learn how to combine these two technologies ArgoCD and Tekton to build fully automated CI/CD pipeline which can be triggerred by Github to manage OpenFaaS functions based on GitOps principles in your local environment using a Cloud Native Tunnel called Inlets
```

Comment:
Too much jargon.

> Learn how to trigger builds of your functions from GitHub using Tekton. Then connect them to Argo CD so that you can deploy as new versions are created.